frontend/src/scenes/feature-flags/featureFlagsLogic.ts

```diff
@@ -68,7 +76,39 @@ export const featureFlagsLogic = kea<featureFlagsLogicType>({
             },
             deleteFlag: (state, { id }) => state.filter((flag) => flag.id !== id),
         },
+        activeTab: [
+            FeatureFlagsTabs.OVERVIEW,
+            {
+                setActiveTab: (state, { tabKey }) =>
+                    Object.values<string>(FeatureFlagsTabs).includes(tabKey) ? tabKey : state,
+            },
+        ],
+        historyPage: [null as number | null, { setHistoryPage: (_, { page }) => page }],
     },
+    actionToUrl: ({ values }) => ({
+        setActiveTab: () => {
+            const searchParams = {
+                ...router.values.searchParams,
+                tab: values.activeTab,
+            }
+            if (values.activeTab !== FeatureFlagsTabs.HISTORY) {
+                delete searchParams['page']
+            }
+            return [router.values.location.pathname, searchParams, router.values.hashParams, { replace: true }]
+        },
+    }),
+    urlToAction: ({ actions, values }) => ({
+        [urls.featureFlags()]: async (_, searchParams) => {
+            const tabInURL = searchParams['tab']
+            if (tabInURL && tabInURL !== values.activeTab) {
+                actions.setActiveTab(tabInURL)
+            }
+            const pageInURL = searchParams['page']
```

Comment:
Weirdly backward/forward navigation doesn't update the page, not sure why.

Reply:
It does now, I don't really understand the fix but I'll take it 🤣